domains/partofart.domain.json

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "Testing Website",
+    "domain": "is-not-a.dev",
+    "subdomain": "example",
+
+    "owner": {
+        "repo": "https://github.com/Malek13X/GCC-Website",
+        "email": "malek13x@gmail.com"
+    },
+
+    "record": {
+        "A": ["1.1.1.1", "1.0.0.1"],
+        "AAAA": ["::1", "::2"],
+        "CNAME": "partofart.com",
+        "MX": ["mx1.partofart.com", "mx2.partofart.com"],
+        "TXT": ["partofart_verification=1234567890"],
+        "CAA": [
+            { "flags": 0, "tag": "issue", "value": "letsencrypt.org" },
+            { "flags": 0, "tag": "issuewild", "value": "sectigo.com" }
+        ],
+        "SRV": [
+            { "priority": 10, "weight": 60, "port": 5060, "target": "sipserver.partofart.com" },
+            { "priority": 20, "weight": 10, "port": 5061, "target": "sipbackup.partofart.com" }
+        ]
```

Comment:
Invalid, more than half of these are example records, they need to be valid records.